### PR TITLE
Task285-Add_curly_braces

### PR DESCRIPTION
**Pull request by tsvetaevAlex**
Task:https://github.com/jdi-testing/jdi-light-csharp/issues/285
Coadcy Pattern: Add curly braces around the nested statement(s) in this 'if' block.
**NOTE:** 
**by mistake changes in file
JDI.Light/JDI.Light.Tests/JDI.Light.Tests.csproj
also getting in current commit**
Sorry.

-------------------------------------✄ cut line ----------------------------------------------
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

**Code formatting**

**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs [hunk 1]
```diff
@@ -23,7 +23,12 @@ public void SetUp()
         [Test]
         public void CheckSingleTest()
         {
-            Assert.DoesNotThrow(() => TestSite.MetalsColorsPage.CbWater.Check(true));
+            /*
+            5/19/2020: update Alex T remove true
+            CbWater.Check(true));
```
Comment (codacy-review) [inline JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs:28]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759&bid=18206213)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759&bid=18206213)

File: JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs [hunk 2]
```diff
@@ -23,7 +23,12 @@ public void SetUp()
         [Test]
         public void CheckSingleTest()
         {
-            Assert.DoesNotThrow(() => TestSite.MetalsColorsPage.CbWater.Check(true));
+            /*
+            5/19/2020: update Alex T remove true
+            from line30: CbWater.Check(true));
```
Comment (codacy-review) [inline JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs:28]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759&bid=18206213)

File: JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs [hunk 3]
```diff
@@ -23,7 +23,12 @@ public void SetUp()
         [Test]
         public void CheckSingleTest()
         {
-            Assert.DoesNotThrow(() => TestSite.MetalsColorsPage.CbWater.Check(true));
+            /*
+            5/19/2020: update Alex T in line 30: remove true
+            Reason: true used as default value
+            */ 
+            CbWater.Check();
```
Comment (v3rde) [inline JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs:30]: Why call `CbWater.Check();` twice?

File: JDI.Light/JDI.Light.Tests/Tests/Composite/SiteDomainTests.cs [hunk 4]
```diff
@@ -35,6 +36,9 @@ public void TestBrokenDomain()
         [TearDown]
         public override void TestTearDown()
         {
+            //not implemented yet
+            //required, will be implemented for using in future
```
Comment (v3rde) [inline JDI.Light/JDI.Light.Tests/Tests/Composite/SiteDomainTests.cs:40]: Unnecessary comment

File: JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs [hunk 5]
```diff
@@ -23,7 +23,12 @@ public void SetUp()
         [Test]
         public void CheckSingleTest()
         {
-            Assert.DoesNotThrow(() => TestSite.MetalsColorsPage.CbWater.Check(true));
+            /*
+            5/19/2020: update Alex T in line 30: remove true
+            Reason: true used as default value
```
Comment (v3rde) [inline JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs:28]: Unnecessary comment

File: JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs (hunk 3 above)
Comment (tsvetaevAlex) [reply JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs:30]: fixed new PR sent

File: JDI.Light/JDI.Light.Tests/Tests/Composite/SiteDomainTests.cs (hunk 4 above)
Comment (tsvetaevAlex) [reply JDI.Light/JDI.Light.Tests/Tests/Composite/SiteDomainTests.cs:40]: fixed new PR sent

File: JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs (hunk 5 above)
Comment (tsvetaevAlex) [reply JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs:28]: fixed new PR sent

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759&bid=18206213)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759&bid=18206213)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759&bid=18206213)

File: JDI.Light/JDI.Light.Tests/Tests/Composite/SiteDomainTests.cs (hunk 4 above)
Comment (tsvetaevAlex) [reply JDI.Light/JDI.Light.Tests/Tests/Composite/SiteDomainTests.cs]: comment removed

File: JDI.Light/JDI.Light.Tests/Tests/Composite/SiteDomainTests.cs [hunk 6]
```diff
@@ -2,6 +2,7 @@
 using JDI.Light.Tests.UIObjects;
 using NUnit.Framework;
 
+
```
Comment (v3rde) [inline JDI.Light/JDI.Light.Tests/Tests/Composite/SiteDomainTests.cs:5]: Remove extra space

File: JDI.Light/JDI.Light/Factories/WebDriverFactory.cs [hunk 7]
```diff
@@ -211,8 +217,11 @@ private string GetDriverName(string driverName)
         public string RegisterDriver(string driverName, Func<IWebDriver> driver)
         {
             if (Drivers.ContainsKey(driverName))
+            {
                 throw Jdi.Assert.Exception(
                     $"Can't register WebDriver {driverName}. Driver with the same name already registered");
+
```
Comment (v3rde) [inline JDI.Light/JDI.Light/Factories/WebDriverFactory.cs:223]: Remove empty line

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759&bid=18206213)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759&bid=18206213)

**Pull request merged**